ProjectForce/force-app/main/default/aura/newD3Comp/newD3CompController.js

```diff
@@ -343,15 +350,20 @@ myAction : function(component, event, helper) {
             helper.setInitFilterValues(component, event);
             //pass the results to the chart creator.
             var ctx = component.find("myChart").getElement();
-            var newChart = helper.createChart(ctx, barOptions_stacked, response.getReturnValue(),
-                                                component.get('v.UserColors'),  component.find('select').get('v.value'));
+            var newChart = helper.createChart(
```

Comment:
What's this doing?

Reply:
put params on different lines for readibility